addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -670,6 +681,7 @@ export const analyticDistribution = {
         business_domain_compute: attrs.business_domain_compute,
         force_applicability: options.force_applicability,
         allow_save: !options.disable_save,
+        company_field: options.company_field || "company_id",
```

Comment:
This means that all records using this widget need to have a `company_id` field 🤔 I'm not sure we want that